<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notes</class>
 <widget class="QWidget" name="Notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:horizontal
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    height: 15px;
    margin: 0px 16px 0 16px;
}

QScrollBar::handle:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;
}

QScrollBar::add-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    width: 15px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;
}

QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 16px;
	border: 1px solid #2d2d2d;
    margin: 16px 0px 16px 0px;
}

QScrollBar::handle:vertical
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    min-height: 20px;
}

QScrollBar::add-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1,     			stop:0 rgba(97, 97, 97, 255),stop:1 rgba(90, 90, 90, 255));
	border: 1px solid #2d2d2d;
    height: 15px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="LeftFrameStyleSheet">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background:rgba(64,64,64,60%);
    border: 1px solid black;
}

QPushButton
{
	background-color: rgba(136, 136, 136, 30%);
	color: white;
	min-width: 80px;
	border-style: solid;
	border-width: 1px;
	border-radius: 2px;
	border-color: black;
	padding: 1px;
}

QPushButton::flat
{
	background-color: transparent;
	border: none;
	color: #fff;
}

QPushButton::hover
{
	background-color: rgba(0, 39, 255, 20%);
	border: 1px solid black;

}

QPushButton:pressed 
{
    background-color: rgb(64, 64, 64);
}

QLabel
{
	color: white;
}

QListView
{
	color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="buttonVertLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="toolNameLabel">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Notes Tool</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backPushButton">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="dirStatusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{
	color: white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">QFrame
{
	background:rgba(64,64,64,60%);
    border: 1px solid black;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="colorButtonLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="whitePushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color: rgba(255, 255, 255, 60%);
	color: black;
	border: 1px solid black;
	padding: 2px;
}

QPushButton:pressed 
{
    background-color: rgb(255, 255, 255);
}
</string>
            </property>
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="redPushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color: rgba(255, 0, 0, 60%);
	color: white;
	border: 1px solid black;
	padding: 2px;
}

QPushButton:pressed 
{
    background-color: rgb(255, 0, 0);
}
</string>
            </property>
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="greenPushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color: rgba(0, 255, 0, 60%);
	color: white;
	border: 1px solid black;
	padding: 2px;
}

QPushButton:pressed 
{
    background-color: rgb(0, 255, 0);
}
</string>
            </property>
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bluePushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	background-color: rgba(0, 0, 255, 60%);
	color: white;
	border: 1px solid black;
	padding: 2px;
}

QPushButton:pressed 
{
    background-color: rgb(0, 0, 255);
}
</string>
            </property>
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit
{
	background:rgba(64,64,64,60%);
    border: 1px solid black;
	color: white;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
